nmdc/pass.go

```diff
@@ -29,3 +30,11 @@ type GetPass struct {
 func (*GetPass) Type() string {
 	return "GetPass"
 }
+
+type LogedIn struct {
+	String
```

Comment:
Is it a `String` or a `Name`?

Reply:
I've checked and it should be a Name since the field contains a username, thanks